Neos.Neos/Resources/Public/JavaScript/Services/index.js

```diff
@@ -0,0 +1,7 @@
+import Configuration from "./Configuration";
+import Notification from "./Notification";
+import Localisation from "./Localisation";
```

Comment:
Should be `Localization` everywhere as our L10n stuff in the Neos core is also written like that. 
https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-ise,_-ize_(-isation,_-ization)